lib_test/crowbar_tests.ml

```diff
@@ -76,34 +73,24 @@ let ds_value =
    *)
 let ds =
   let open Rrd in
-  Cb.choose [
-    Cb.map Cb.[ds_value; float; float] (fun v min max ->
-      let min, max = castd2s min max in
-      Cb.guard (min < max); ds_create "derive" ~min ~max Derive v);
-    Cb.map Cb.[ds_value; float; float] (fun v min max ->
-      let min, max = castd2s min max in
-      Cb.guard (min < max); ds_create "absolute" ~min ~max Absolute v);
-    Cb.map Cb.[ds_value; float; float] (fun v min max ->
-      let min, max = castd2s min max in
-      Cb.guard (min < max); ds_create "gauge" ~min ~max Gauge v)
-  ]
+  let ds_type = Cb.(choose [const Derive; const Absolute; const Gauge]) in
+  Cb.(map [ds_value; float; float; ds_type] (fun v x y typ ->
```

Comment:
I was wondering why this compiles, looks like they overrode the syntax for list inside the Crowbar module.